client/src/App.js

```diff
@@ -16,9 +17,15 @@ function App() {
           <Route path="/addTour">
             <AddTour />
           </Route>
+          <Route path="/addConcurrentTour">
+            <AddTour concurrentTour={true} />
```

Comment:
Boolean properties have a short form:

```suggestion
            <AddTour concurrentTour />
```